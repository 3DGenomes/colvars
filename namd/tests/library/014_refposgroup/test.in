colvarsTrajFrequency    1
colvarsRestartFrequency 2000

colvar {
  name r

  outputAppliedForce yes

  rmsd {
    refPositionsFile ../Common/310turn-ideal.pdb
    debugGradients yes
    atoms {
      atomNumbers  4
      rotateReference yes
      centerReference yes
      fittingGroup {
        atomNumbers  54 64 74 84 99 
      }
      refPositionsFile ../Common/310turn-ideal.pdb # ref prot positions for fitting
    }
  }
}

harmonic {
  colvars r
  centers 0.
  forceConstant 0.1
}

