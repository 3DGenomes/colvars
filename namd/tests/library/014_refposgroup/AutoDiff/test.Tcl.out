TCL: TOTAL FORCE: ( 1.3848077684563975e-9 3.8780578748287553e-10 9.441097015283617e-9 )
TCL: TOTAL TORQUE: ( 9.617198593048215e-9 -6.671102781297122e-8 1.3296061474044052e-9 )
TCL: TOTAL FORCE: ( 1.3845900537212685e-9 3.8764050303008446e-10 9.44052769291659e-9 )
TCL: TOTAL TORQUE: ( 9.585253035737651e-9 -6.669778018775219e-8 1.3328810277712932e-9 )
TCL: TOTAL FORCE: ( 1.3847469559902237e-9 3.8763073306746776e-10 9.440896564516521e-9 )
TCL: TOTAL TORQUE: ( 9.551187840628472e-9 -6.668942420517965e-8 1.3372525309307548e-9 )
TCL: TOTAL FORCE: ( 1.384702519313663e-9 3.881029941865677e-10 9.441947834698539e-9 )
TCL: TOTAL TORQUE: ( 9.519129484658606e-9 -6.66852830732978e-8 1.3450229818801063e-9 )
TCL: TOTAL FORCE: ( 1.3846691848673487e-9 3.885954891202914e-10 9.443014370447145e-9 )
TCL: TOTAL TORQUE: ( 9.487003183039633e-9 -6.668641727713975e-8 1.3531336051642029e-9 )
TCL: TOTAL FORCE: ( 1.3843890478426601e-9 3.892626498913643e-10 9.44320938112142e-9 )
TCL: TOTAL TORQUE: ( 9.455817462367122e-9 -6.66831190265782e-8 1.3625300332442691e-9 )
TCL: TOTAL FORCE: ( 1.3839316637120902e-9 3.896168387917953e-10 9.442643389423466e-9 )
TCL: TOTAL TORQUE: ( 9.423963831522997e-9 -6.667165219909066e-8 1.3697731282569237e-9 )
TCL: TOTAL FORCE: ( 1.38348035805258e-9 3.898123490664318e-10 9.441210147009826e-9 )
TCL: TOTAL TORQUE: ( 9.39312005954207e-9 -6.665373808445452e-8 1.3755932504189161e-9 )
TCL: TOTAL FORCE: ( 1.3832543721559176e-9 3.898995015738649e-10 9.439174442071874e-9 )
TCL: TOTAL TORQUE: ( 9.3661967071057e-9 -6.663279261687194e-8 1.379807823553847e-9 )
TCL: TOTAL FORCE: ( 1.3832883172248955e-9 3.9039815824537527e-10 9.436075421032086e-9 )
TCL: TOTAL TORQUE: ( 9.336885931077177e-9 -6.660509788147806e-8 1.3868997061905475e-9 )
TCL: TOTAL FORCE: ( 1.3836642387410336e-9 3.9107186933229343e-10 9.431551428740192e-9 )
TCL: TOTAL TORQUE: ( 9.307737691699458e-9 -6.656929540937995e-8 1.3947443755490951e-9 )
TCL: TOTAL FORCE: ( 1.3839572821083834e-9 3.916390822755744e-10 9.426975866588805e-9 )
TCL: TOTAL TORQUE: ( 9.281833079910484e-9 -6.653200035344753e-8 1.4013922244870969e-9 )
TCL: TOTAL FORCE: ( 1.383979764124632e-9 3.921172830878561e-10 9.422768287858929e-9 )
TCL: TOTAL TORQUE: ( 9.256831745574345e-9 -6.64963772933902e-8 1.4075576260097478e-9 )
TCL: TOTAL FORCE: ( 1.3838280799038927e-9 3.92610777222302e-10 9.4177918796845e-9 )
TCL: TOTAL TORQUE: ( 9.234643272293397e-9 -6.645546868355723e-8 1.413493377899755e-9 )
TCL: TOTAL FORCE: ( 1.3835779466564446e-9 3.927674852022278e-10 9.412744750303403e-9 )
TCL: TOTAL TORQUE: ( 9.216621243979262e-9 -6.640994421047708e-8 1.4163542561007603e-9 )
TCL: TOTAL FORCE: ( 1.3833544865171632e-9 3.930113179340111e-10 9.407115864057403e-9 )
TCL: TOTAL TORQUE: ( 9.198239947494358e-9 -6.635960270173769e-8 1.419735384811105e-9 )
TCL: TOTAL FORCE: ( 1.3832190670637345e-9 3.935571868396437e-10 9.401294909228142e-9 )
TCL: TOTAL TORQUE: ( 9.177917537073199e-9 -6.630868298884707e-8 1.4254650237965905e-9 )
TCL: TOTAL FORCE: ( 1.3833439671540049e-9 3.9406269913833114e-10 9.39491939799808e-9 )
TCL: TOTAL TORQUE: ( 9.155737501487238e-9 -6.62538384155198e-8 1.4308423335052112e-9 )
TCL: TOTAL FORCE: ( 1.3834061118878083e-9 3.9413142194355544e-10 9.389531985259936e-9 )
TCL: TOTAL TORQUE: ( 9.13508824140763e-9 -6.620808434831815e-8 1.4332008912987249e-9 )
TCL: TOTAL FORCE: ( 1.3834735579365542e-9 3.93653998287391e-10 9.384015453584027e-9 )
TCL: TOTAL TORQUE: ( 9.116665200537e-9 -6.615981806845639e-8 1.4313073504190754e-9 )
